apax/config/train_config.py

```diff
@@ -175,7 +175,9 @@ class ModelConfig(BaseModel, extra="forbid"):
     emb_init: Optional[str] = "uniform"
 
     nn: List[PositiveInt] = [512, 512]
+    w_init: Literal["normal", "lecun"] = "normal"
     b_init: Literal["normal", "zeros"] = "normal"
+    use_ntk: bool = True
```

Comment:
I would unify `use_ntk` and `is_ntk` variable names. Personally, I think it could just be `ntk` but the boolean type is more clear with `use/is`